lib/dry/cli/command.rb

```diff
@@ -108,6 +109,46 @@ def self.example(*examples)
         @examples += examples.flatten(1)
       end
 
+      # Add extra free text when `--help` flag is used.
+      #
+      # @param blk [Proc] a block that must return a free form string to add
+      #   extra information
+      #
+      # @since 0.8.0
+      #
+      # @example
+      #   require "dry/cli"
+      #
+      #   class Convert < Dry::CLI::Command
+      #     help do
```

Comment:
Have you though to support just `help "String"` too?

Reply:
@rngtng Yeah, that could work too.